<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_well_log</class>
 <widget class="QWidget" name="Form_well_log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Well Logging</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="PlotWidget" name="widget_graph_well_log"/>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_add_dir_well_log">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(180, 255, 205);</string>
         </property>
         <property name="text">
          <string>ADD DIR</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_rm_all_well_db">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 162, 190);</string>
         </property>
         <property name="text">
          <string>REMOVE ALL</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_rm_well_log">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 185, 185);</string>
         </property>
         <property name="text">
          <string>REMOVE</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QListWidget" name="listWidget_well_log"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_add_well_log">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(191, 255, 191);</string>
         </property>
         <property name="text">
          <string>ADD</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBox_raar_signal">
         <property name="text">
          <string>radar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
